doc/user/upgrade/upgrade_guide.md

```diff
@@ -22,6 +22,81 @@ $ kubectl edit deployment/etcd-operator
 ### Incompatible upgrade
 In the case of an incompatible upgrade, the process requires restoring a new cluster from backup. See the [incompatible upgrade guide](incompatible_upgrade.md) for more information.
 
+
+## v0.5.x -> v0.6.0
+
+### Breaking Change
+
+The v0.6.0 release removes [operator level s3 backup](https://github.com/coreos/etcd-operator/blob/v0.5.1/doc/user/backup_config.md#operator-level-configuration).
+
+If you are using operator level s3 backup, You must migrate your cluster to use [cluster level backup](../backup_config.md#s3-on-aws) for v0.6.0.
+
+Steps to upgrade:
+
+Assuming your cluster uses [operator level s3 backup](https://github.com/coreos/etcd-operator/blob/v0.5.1/doc/user/backup_config.md#operator-level-configuration).
```

Comment:
Remove this line.

> Assuming your cluster uses 

Not a good manner to assume in upgrade docs.. Just tell users that they need to do something only if what condition is met.